### PR TITLE
This commit fixes #105

### LINKED ISSUE #105: tidy up UI on share modal and impact makers block
The share buttons should be the same width on all devices, regardless of inner content. This could be resolved with css grid. 
Screenshot 2019-06-05 15 38 56

These links should all start from the same place; one under the other 
Screenshot 2019-06-05 15 39 18

### PR DESCRIPTION
**Pull request by patyiscoding**
Changed the social section from a list to CSS grid; evened out the width of all the buttons; evened out the contributor links

### CODE REVIEW

File: src/components/Footer/Styles.module.scss [hunk 1]
```diff
@@ -200,7 +200,7 @@
   margin: 13px auto 30px;
   display: flex;
   flex-direction: column;
-  align-items: center;
+  //align-items: center;
```
Comment (rachel-harris1) [inline src/components/Footer/Styles.module.scss]: feel free to remove these as they are not serving any purpose atm

File: src/components/ShareSocialCta/Styles.module.scss [hunk 2]
```diff
@@ -128,28 +128,44 @@
...
-  align-items: center;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  flex-direction: column;
-  list-style-type: none;
-  padding: 0;
-  margin: 0;
-  @include mq("tablet") {
-    flex-direction: row;
+  display: grid;
```
Comment (rachel-harris1) [inline src/components/ShareSocialCta/Styles.module.scss:131]: amazing, thank you for implementing css grid! 🎉 it is soooo much cleaner!

File: src/components/ShareSocialCta/Styles.module.scss [hunk 3]
```diff
@@ -128,28 +128,44 @@
...
-    margin-right: 0;
-  }
-  @include mq("tablet") {
-    margin: 10px 10px 0 0;
-  }
-}
+  grid-column: auto;
+  grid-row: auto;
+}
+//.socialContainer {
```
Comment (rachel-harris1) [inline src/components/ShareSocialCta/Styles.module.scss]: it's probably best to remove these styles if they are no longer needed

File: src/components/SocialButtons/styles.module.scss [hunk 4]
```diff
@@ -5,13 +5,15 @@
 .button {
   background-color: $white;
   border: 2px solid $grayBorder;
-  max-width: max-content;
-  height: 30px;
+  //max-width: max-content;
```
Comment (rachel-harris1) [inline src/components/SocialButtons/styles.module.scss]: same here please :)

**Review by rachelfranklin1 — COMMENTED** [review]
@lauvrenn thanks for contributing to Climate Choice & congrats on your first PR here 🎉  

This code is such a big improvement & it looks great locally! 

Would you mind removing the commented out code? We no longer need it & may as well reduce the number of lines of code we have in Climate Choice. 

Once that is done, we can get it merged :D

**Comment by patyiscoding** [comment]
@rachelfranklin1 Thanks for the warm welcome! Just updated the files, so all comments should be removed :)

**Review by rachelfranklin1 — APPROVED** [review]
nice one @lauvrenn ! 🎉

**Pull request merged**